frontend/packages/helm-plugin/console-extensions.json

```diff
@@ -74,5 +74,86 @@
     "flags": {
       "required": ["OPENSHIFT_HELM"]
     }
+  },
+  {
+    "type": "topology.details/resource-link",
+    "properties": {
+      "priority": 1000,
+      "link": { "$codeRef": "helmTopology.getHelmReleasePanelResourceLink" }
+    },
+    "flags": {
+      "required": ["OPENSHIFT_HELM"]
+    }
+  },
+  {
+    "type": "topology.details/tab",
+    "properties": {
+      "id": "helm-release-panel-tab-details",
+      "label": "%helm-plugin~Details%",
+      "insertBefore": "helm-release-panel-tab-resource"
+    },
+    "flags": {
+      "required": ["OPENSHIFT_HELM"]
+    }
+  },
+  {
+    "type": "topology.details/tab",
+    "properties": {
+      "id": "helm-release-panel-tab-resource",
+      "label": "%helm-plugin~Resources%",
+      "insertBefore": "helm-release-panel-tab-releaseNotes",
+      "insertAfter": "helm-release-panel-tab-details"
+    },
+    "flags": {
+      "required": ["OPENSHIFT_HELM"]
+    }
+  },
+  {
+    "type": "topology.details/tab",
+    "properties": {
+      "id": "helm-release-panel-tab-releaseNotes",
+      "label": "%helm-plugin~Release notes%",
+      "insertAfter": "helm-release-panel-tab-resource"
+    },
+    "flags": {
+      "required": ["OPENSHIFT_HELM"]
+    }
+  },
+  {
+    "type": "topology.details/tab-section",
+    "properties": {
+      "id": "helm-release-panel-tab-section-details",
+      "tab": "helm-release-panel-tab-details",
+      "section": { "$codeRef": "helmTopology.getHelmReleasePanelDetailsTabSection" },
+      "insertBefore": "helm-release-panel-tab-section-resource"
+    },
+    "flags": {
+      "required": ["OPENSHIFT_HELM"]
+    }
+  },
+  {
+    "type": "topology.details/tab-section",
+    "properties": {
+      "id": "helm-release-panel-tab-section-resource",
+      "tab": "helm-release-panel-tab-resource",
+      "section": { "$codeRef": "helmTopology.getHelmReleasePanelResourceTabSection" },
+      "insertAfter": "helm-release-panel-tab-section-details",
+      "insertBefore": "helm-release-panel-tab-section-releaseNotes"
```

Comment:
Why both before and after properties?

Note that the order in which they are defined should remain in tact so long as they are defined by the same plugin. 
Therefore if the helm plugin is contributing all help sections, we should not even need to define the before or after properties.